### ISSUE TITLE
Add ReSharper 9 Support

### ISSUE THREAD

**Issue by jpickwell** [JakeGinnivan/XamlAttributeOrderingCodeCleanup#8]

**Comment by jerriep** [comment]
+1 

Any chance of this happening?

**Comment by JakeGinnivan** [comment]
Would love a pull request. I have a lot on my plate at the moment

**Comment by jerriep** [comment]
Struggling to get around to everything in my own open source projects as well, so I cannot help with this one unfortunately :(

**Comment by JakeGinnivan** [comment]
Yeah, I am working on a release for https://github.com/ParticularLabs/GitVersion, https://github.com/shouldly/shouldly and working with a few people to get https://github.com/GitTools/GitReleaseNotes sorted. 

Will see what I can do but no promises

**Comment by jerriep** [comment]
No worries, I understand your predicament as I find myself in the same spot. If you get around to it someday it is cool, otherwise it is not a problem.

**Comment by JakeGinnivan** [comment]
Hey @citizenmatt 

How is your bandwidth? :P

**Comment by jerriep** [comment]
Oh BTW, Shouldly looks really cool. Have been using FluentAssertions mostly, but will look into this as well
